experimental/dds/tree2/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -179,6 +179,7 @@ export type NodeChangeInverter = (
 export type NodeChangeRebaser = (
 	change: NodeChangeset | undefined,
 	baseChange: NodeChangeset | undefined,
+	deleted?: boolean,
```

Comment:
Some docs would be nice:
- When should true be passed? When the change deletes the node, when the base change deletes the node, either, both?
- How is it interpreted when not passed in?

Reply:
Added this doc comment:

```
True when the baseChange deletes the node. Defaults to false if not specified.
```